_posts/2020-10-12-sparklyr-flint-0.2.0-released/sparklyr-flint-0.2.0.Rmd

```diff
@@ -0,0 +1,143 @@
+---
+title: "sparklyr.flint 0.2: ASOF Joins, OLS Regression, and Additional Summarizers"
```

Comment:
Seems like in general, we (I ;-)) have introduced some tradition of not capitalizing titles as a whole ... to me personally this one would look better "without", but up to you...

Reply:
OK. Will fix it replacing "Additional Summarizers" with "additional summarizers"